### DISCUSSION TITLE
Announcing version 3.0.2

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1171] [Announcements]
We are pleased to announce version 3.0.2, which is only bug fixes for version 3.0.0

Release 3.0.1 contains a couple of problems, so please do not try to use that.

The 3.0.2 release have had contributions from no less than 10 contributors, a BIG THANK YOU to all that helped iron out the bugs of 3.0.0

Next up is 3.1.0 which again are expected to contain important changes, this time concentrating on performance.

**Comment by serroba** [comment]
Thank you @janiversen really cool to see how active the project is
